File: DheerajKumar/problem27.cpp
//27. WAP to input a lower limit and an upper limit from user and print all the even numbers in between the given limits.

#include<iostream>
using namespace std;

void printNumebers(int lowerLimit, int upperLimit){
    cout<<"[ - ";
    for(int i = lowerLimit + 1; i < upperLimit; i++){
        if(i % 2 == 0){
            cout<<i<<" - ";
        }
    }
    cout<<" ]";
}

int main(){
    int lowerLimit, upperLimit;
    cout<<"Enter lower case ->> ";
    cin>>lowerLimit;
    cout<<"Enter upper case ->> ";
    cin>>upperLimit;

    printNumebers(lowerLimit, upperLimit);
}

File: LovelyKumari/BasicProgramming/Problem27.java
import java.util.Scanner;

public class Problem27 {
    //WAP to input a lower limit and an upper limit from user and print all the even numbers in between the given limits
    public static void main(String[] args) {
        Scanner sc=new Scanner(System.in);
        System.out.print("Enter the lower limit: ");
        int lowerLimit=sc.nextInt();
        System.out.print("Enter the upper limit: ");
        int upperLimit=sc.nextInt();

        //Printing all the even numbers in the range of lowerLimit to upperLimit

        System.out.println("Even numbers "+ lowerLimit+ " and " + upperLimit+ " are:");
        if(lowerLimit % 2 !=0){
            lowerLimit++ ;
        } 
        for(int i=lowerLimit;i<=upperLimit ; i+=2){
            System.out.println(i+" ");

        }
    }
}
